snappy/binaries.go

```diff
@@ -112,7 +111,7 @@ ubuntu-core-launcher {{.UdevAppName}} {{.AaProfile}} {{.Target}} "$@"
 		SnapArch:    arch.UbuntuArchitecture(),
 		SnapPath:    pkgPath,
 		Version:     m.Version,
-		UdevAppName: udevPartName,
+		UdevAppName: fmt.Sprintf("%s.%s", m.Name, app.Name),
```

Comment:
I wonder if we should use "_" here for the decliner? This would allow us to still have the full snapname.origin_appname ID style string.